db/migrate/20190902182932_set_default_ocr_correction_value.rb

```diff
@@ -0,0 +1,8 @@
+class SetDefaultOcrCorrectionValue < ActiveRecord::Migration
+  def change
+    # Set default value to false
+    change_column :works, :ocr_correction, :boolean, default: false
+    # Retroactively set NULL to false
+    Work.where(ocr_correction: nil).update_all(ocr_correction: false)
```

Comment:
Check me on this, but isn't this unnecessary once the default is set on the column?

Reply:
I was mistaken: in MySQL/MariaDB, this sets the default for new rows but does not update values in existing rows.